webapp/channels/babel.config.js

```diff
@@ -41,17 +41,28 @@ const config = {
     sourceType: 'unambiguous',
 };
 
-// Jest needs module transformation
 config.env = {
     test: {
-        presets: config.presets,
+        presets: config.presets.map((preset) => {
+            const [presetName, presetOptions] = preset;
+            if (presetName === '@babel/preset-env') {
+                return [
+                    presetName,
+                    {
+                        ...presetOptions,
+                        modules: 'auto', // Jest needs module transformation
+                    },
+                ];
+            }
+
+            return preset;
```

Comment:
![giphy](https://media0.giphy.com/media/eJAH5DRebLCxAvBhyU/giphy.gif?cid=ecf05e47q4oa3j0i7xxaqx9rpxrtbdm4uydc3rdko03yntqz&ep=v1_gifs_search&rid=giphy.gif&ct=g)

Reply:
That's a good catch, but I think that the mutation actually meant that `modules` was set to `auto` for both test and production builds before.

Could you actually just change that above instead of us having to modify `config.presets` at all?

Reply:
Yeah, I just checked and both are set to `auto` currently. I don't entirely understand the purpose of that since it probably doesn't affect the production build because Webpack is handling that

Reply:
Yes i just check the docs, it recommends setting 'auto' https://babeljs.io/docs/babel-preset-env#modules